lib/gltfPrimitiveToCesiumGeometry.js

```diff
@@ -7,9 +7,39 @@ var Geometry = Cesium.Geometry;
 var GeometryAttribute = Cesium.GeometryAttribute;
 var ComponentDatatype = Cesium.ComponentDatatype;
 var defined = Cesium.defined;
+var Cartesian2 = Cesium.Cartesian2;
+var Cartesian3 = Cesium.Cartesian3;
+var Cartesian4 = Cesium.Cartesian4;
 
 module.exports = gltfPrimitiveToCesiumGeometry;
 
+// Helper function for packing Cartesians to arrays
+function packArray(values) {
+    var length = values.data.length;
+    var packed;
+    var i;
+    switch (values.type) {
+        case 'Cartesian2':
+            packed = new Array(length * 2);
+            for (i = 0; i < length; ++i) {
+                Cartesian2.pack(values.data[i], packed, i * 2);
+            }
+            return packed;
+        case 'Cartesian3':
+            packed = new Array(length * 3);
+            for (i = 0; i < length; ++i) {
+                Cartesian3.pack(values.data[i], packed, i * 3);
+            }
+            return packed;
+        case 'Cartesian4':
+            packed = new Array(length * 4);
+            for (i = 0; i < length; ++i) {
+                Cartesian4.pack(values.data[i], packed, i * 4);
+            }
+            return packed;
+    }
+}
```

Comment:
Now that Cesium 1.23 is on npm, we can use the `CartesianX.packArray` functions instead.